### PR TITLE
[trivial] Fixed -Wshadow warnings

### PR DESCRIPTION
**Pull request by tjps**

### CODE REVIEW

File: src/test/checkqueue_tests.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ struct MemoryCheck {
     };
     ~MemoryCheck(){
         fake_allocated_memory -= b;
-    
+
```
Comment (practicalswift) [inline src/test/checkqueue_tests.cpp]: Intentional blank line? :-)

File: src/util.h [hunk 2]
```diff
@@ -94,9 +94,9 @@ template<typename... Args> std::string FormatStringFromLogArgs(const char *fmt,
 } while(0)
 
 template<typename... Args>
-bool error(const char* fmt, const Args&... args)
+bool error(const char* fmt, Args... args)
```
Comment (laanwj) [inline src/util.h]: Not sure what you're doing here but I doubt it is required to "Fix Wshadow".

**Comment by laanwj** [comment]
Also may be duplicate of #9911.

**Comment by laanwj** [comment]
Closing in favor of #10009 which is cleaner and more targeted.

**Pull request closed without merging**